package.json

```diff
@@ -4,7 +4,7 @@
     "homepage": ".",
     "private": true,
     "dependencies": {
-        "@gridsuite/commons-ui": "^0.10.0",
+        "@gridsuite/commons-ui": "^0.12.2",
```

Comment:
It's not the scope of this PR. There is already a PR to update the comm-ui version used by gridexplore